examples/sgd/sgd.go

```diff
@@ -4,25 +4,47 @@ import (
 	torch "github.com/wangkuiyi/gotorch"
 )
 
+// Net struct
+type Net struct {
+	l1, l2 *torch.Linear
+}
+
+// NewNet creats a net instance
+func NewNet(m *torch.Model) *Net {
+	return &Net{
+		l1: torch.NewLinear(m, 100, 200, false),
```

Comment:
Due to 

https://github.com/pytorch/examples/blob/8df8e747857261ea481e0b2492413d52bf7cc3a8/mnist/main.py#L18

could we name it `nn.Linear` rather than `torch.NewLinear`?

Reply:
```go
func Linear(...) Module {
  return &linear{ ...}
}
```

where `Module` names an interface defined in `torch.nn`.